tmva/pymva/inc/TMVA/MethodPyKeras.h

```diff
@@ -76,14 +76,17 @@ namespace TMVA {
       /// Get the Keras backend (can be: TensorFlow, Theano or CNTK)
       EBackendType GetKerasBackend();
       TString GetKerasBackendName();
+      // flag to indicate we are using the Keras shipped with Tensorflow 2
+      Bool_t UseTFKeras() const { return fUseTFKeras; }
 
-    private:
+   private:
 
       TString fFilenameModel; // Filename of the previously exported Keras model
       UInt_t fBatchSize {0}; // Training batch size
       UInt_t fNumEpochs {0}; // Number of training epochs
       Int_t fNumThreads {0}; // Number of CPU threads (if 0 uses default values)
       Int_t fVerbose; // Keras verbosity during training
+      Bool_t fUseTFKeras { kFALSE};   // use Keras from Tensorflow (-1, default, 0 false, 1, true)
```

Comment:
`fUseTFKeras` is a bool but the comments says it can take the value -1

Reply:
Thanks, the comment is wrong !